src/net/download.rs

```diff
@@ -157,12 +157,12 @@ impl Requestable for BlockRequestKey {
     fn make_request_type(&self, peer_host: PeerHost) -> HttpRequestType {
         match self.kind {
             BlockRequestKeyKind::Block => HttpRequestType::GetBlock(
-                HttpRequestMetadata::from_host(peer_host),
+                HttpRequestMetadata::from_host(peer_host, Some(self.sortition_height)),
```

Comment:
`self.sortition_height` is not the Stacks block height; it's the Bitcoin block height, minus the start Bitcoin block height.

Reply:
ohh I see.